php/libraries/Utility.class.inc

```diff
@@ -539,7 +539,7 @@ class Utility
      */
     static function toArray($var)
     {
-        if (!isset($var[0])) {
+        if (!array_key_exists(0, $var)) {
```

Comment:
Does this work if i pass it an array like `array('foo' -> 'bar')`? Couldn't you use `is_array()` instead